src/lib/isTaxID.js

```diff
@@ -1126,23 +1156,23 @@ const taxIdFormat = {
   'sk-SK': /^\d{6}\/{0,1}\d{3,4}$/,
   'sl-SI': /^[1-9]\d{7}$/,
   'sv-SE': /^(\d{6}[-+]{0,1}\d{4}|(18|19|20)\d{6}[-+]{0,1}\d{4})$/,
-
 };
 // taxIdFormat locale aliases
 taxIdFormat['lb-LU'] = taxIdFormat['fr-LU'];
 taxIdFormat['lt-LT'] = taxIdFormat['et-EE'];
 taxIdFormat['nl-BE'] = taxIdFormat['fr-BE'];
+taxIdFormat['fr-CA'] = taxIdFormat['en-CA'];
 
 // Algorithmic tax id check functions for various locales
 const taxIdCheck = {
-
   'bg-BG': bgBgCheck,
   'cs-CZ': csCzCheck,
   'de-AT': deAtCheck,
   'de-DE': deDeCheck,
   'dk-DK': dkDkCheck,
   'el-CY': elCyCheck,
   'el-GR': elGrCheck,
+  'en-CA': isCanadianSIN,
```

Comment:
This should have been called `enCaCheck` but we fix this in a later refactor, it doesn't affect the user